tools/release/release_doc_template.md

```diff
@@ -163,17 +126,12 @@ script generates the symbols based on the latest tags.
       branch:
       https://github.com/denoland/apiland_scripts/actions/workflows/release_cli.yml
   - [ ] Verify the workflow ran successfully.
-- [ ] Run the `release STD` workflow in the apiland_scripts repo on the main
-      branch:
-      https://github.com/denoland/apiland_scripts/actions/workflows/release_std.yml
-  - [ ] Verify the workflow ran successfully.
 
   <details>
      <summary>Failure Steps</summary>
 
   1. Clone `deno/apiland_scripts`.
   2. Execute `deno task release:cli`.
-  3. Execute `deno task release:std`.
```

Comment:
We should remove the `release:std` task if it's no longer use:md.

Reply:
We could need it in the future when we need to release `deno.land/std` for some reason (for example, for security fixes)